client/me/purchases/billing-history/utils.tsx

```diff
@@ -229,6 +229,31 @@ function renderSpaceAddOnquantitySummary(
 	} );
 }
 
+export function renderTransactionVolumeSummary(
+	{ volume, wpcom_product_slug }: BillingTransactionItem,
+	translate: LocalizeProps[ 'translate' ]
+) {
+	if ( ! volume ) {
+		return null;
+	}
+
+	volume = parseInt( String( volume ) );
+
+	if ( 'dotblog_domain' !== wpcom_product_slug ) {
+		return null;
+	}
+
+	return translate(
+		'Domain registered for %(quantity)d year',
```

Comment:
This is a good point. Since the receipt is supposed to be a snapshot in time of the purchase as it was at the time of transaction we wouldn't be able to use a dynamic value here like an expiration date or a sum of volumes.

In this case, a renewal would be a separate receipt, so it may make sense to change "Domain registered for 1 year" to "Domain _renewed_ for 1 year" on renewal receipts.

Reply:
Good point.

Reply:
This was fixed.